### PR TITLE
Vcard highlevel interface pr

### PR DESCRIPTION
**Pull request by matthiasbeyer**
This PR contains the high-level interface for Vcard types.

Notice: No tests yet. If you would like to have some, please suggest some,... I'm a bit lost on what to test.

Rel https://github.com/untitaker/rust-vobject/issues/14

### CODE REVIEW

File: src/error.rs [hunk 1]
```diff
@@ -10,6 +10,12 @@ error_chain! {
             description("Parser error")
             display("{}", desc)
         }
+
+        NotAVCard(content: String) {
+            description("Input is not a valid VCard")
+            display("Not a VCard: '{}'", content)
```
Comment (untitaker) [inline src/error.rs]: Not sure including the entire input is a good idea? This will make output potentially very verbose.

File: src/util.rs [hunk 2]
```diff
@@ -0,0 +1,16 @@
+use std::collections::BTreeMap;
```
Comment (untitaker) [inline src/util.rs]: We might as well rename this file to `params.rs`.

**Comment by matthiasbeyer** [comment]
Updated with your requested changes. Very basic tests are in the codebase with this PR.

**Comment by untitaker** [comment]
Awesome, thanks!

**Comment by untitaker** [comment]
Should I release a new version, or are you planning to open more PRs soonish?

**Comment by matthiasbeyer** [comment]
I plan to open more soonish and also I am not sure whether this actually breaks something. I just noticed that `;`-seperated lists contain a `\\` now... Not sure whether this is standard-conform (Also have a look at the last test in this PR).

**Comment by untitaker** [comment]
No that isn't standards conform.

**Comment by untitaker** [comment]
I'll fix it, hang on

**Comment by untitaker** [comment]
Done in https://github.com/untitaker/rust-vobject/commit/50d72dffafb023f27610a7dbe2761673467b5b1c.

**Pull request merged**